main.go

```diff
@@ -71,11 +98,19 @@ func main() {
 		LeaderElection:         enableLeaderElection,
 		LeaderElectionID:       "39f5e5c3.nodefeaturediscoveries.nfd.kubernetes.io",
 	})
+
+	// If the manager could not be started, then log the error as an operator setup error
```

Comment:
Another pretty self-evident case that could probably be dropped

Reply:
Removed now.